src/components/UTIL_PageHeader.component

```diff
@@ -63,6 +63,18 @@
     <apex:attribute name="saveAndNewDisabled" type="Boolean" default="false"
         description="Whether the save and new button should be disabled, defaults to false."/>
 
+    <!-- Continue attributes -->
+    <apex:attribute name="showcontinueBtn" type="Boolean" default="false"
+                    description="Set to false to hide continue button, default is false"/>
+    <apex:attribute name="continueAction" type="ApexPages.Action" default=""
+                    description="Action for continue button."/>
+    <apex:attribute name="continueLabel" type="String"
+                    description="Label for continue button. When blank will display btnContinue label."/>
+    <apex:attribute name="continueImmediate" type="Boolean" default="false"
```

Comment:
description does not match default value.

Reply:
Very good catch. So much for copy/paste. I also fixed other descriptions that were wrong for the same reason.